demos/using-dev-build/webapp/index.js

```diff
@@ -17,38 +17,48 @@ export const init = ({ IDL }) => {
 
 // Autofills the <input> for the II Url to point to the correct canister.
 document.body.onload = () => {
-  const testnetII =
-    process.env.DFX_NETWORK == "local"
-      ? `http://localhost:8000/?canisterId=${process.env.II_CANISTER_ID}`
-      : process.env.DFX_NETWORK == "ic"
-      ? `https://${process.env.II_CANISTER_ID}.ic0.app`
-      : `https://${process.env.II_CANISTER_ID}.dfinity.network`;
-  document.getElementById("iiUrl").value = testnetII;
+  let iiUrl;
+  if (process.env.DFX_NETWORK === "local") {
+    iiUrl = `http://localhost:8000/?canisterId=${process.env.II_CANISTER_ID}`;
+  } else if (process.env.DFX_NETWORK === "ic") {
+    iiUrl = `https://${process.env.II_CANISTER_ID}.ic0.app`;
+  } else {
+    iiUrl = `https://${process.env.II_CANISTER_ID}.dfinity.network`;
+  }
+  document.getElementById("iiUrl").value = iiUrl;
 };
 
 document.getElementById("loginBtn").addEventListener("click", async () => {
-  // When the user clicks, we let agent-js do its thing and redirect open an II
-  // tab for authentication
-  const iiUrl = document.getElementById("iiUrl").value;
+  // When the user clicks, we start the login process.
+  // First we have to create and AuthClient.
   const authClient = await AuthClient.create();
 
-  // This API is just weird
+  // Find out which URL should be used for login.
+  const iiUrl = document.getElementById("iiUrl").value;
+
+  // Call authClient.login(...) to login with Internet Identity. This will open a new tab
+  // with the login prompt. The code has to wait for the login process to complete.
+  // We can either use the callback functions directly or wrap the in a promise as
+  // we have done here.
   await new Promise((resolve, reject) => {
     authClient.login({
       identityProvider: iiUrl,
       onSuccess: resolve,
-      onError: reject,
+      onError: reject
     });
   });
 
-  // At this point, we're authenticated with II and we can query the whoami canister
+  // At this point, we're authenticated with II.
```

Comment:
Decide whether you want periods on all the comments or on none. 

It seems that you have a mix now.